src/routing/router.tsx

```diff
@@ -1,15 +1,18 @@
-import { BrowserRouter, Navigate, useRoutes } from "react-router-dom";
-import { routes } from "./routes";
-import { ConfigsPage } from "../pages/configPages/configs/configsPage";
-import { Layout } from "../layout/layout";
-import { SchemasPage } from "../pages/schemaPages/schemas/schemas";
+import { BrowserRouter, Navigate, useRoutes } from 'react-router-dom';
+import { routes } from './routes';
+
+import { Layout } from '../layout/layout';
+import { ConfigsPage } from '../pages/configs/configsPage';
+import { SchemasPage } from '../pages/schemas/schemas';
+import { NotFoundPage } from '../pages/notFound/notFound';
 
 export const Router: React.FC = () => {
   const Routes: React.FC = () => {
     return useRoutes([
-      { path: "*", element: <Navigate to={routes.NOT_FOUND} /> },
+      { path: '*', element: <Navigate to={routes.NOT_FOUND} /> },
       { path: routes.CONFIG, element: <ConfigsPage /> },
       { path: routes.SCHEMA, element: <SchemasPage /> },
+      { path: routes.NOT_FOUND, element: <NotFoundPage /> },
```

Comment:
you are missing HOME path, right now it redirects to 404.
You can send it to the config page for now.